swap_meet/vendor.py

```diff
@@ -1,2 +1,125 @@
+from swap_meet.item import Item
+from swap_meet.decor import Decor
+from swap_meet.clothing import Clothing
+from swap_meet.electronics import Electronics
+
 class Vendor:
-    pass
\ No newline at end of file
+
+    """
+    Wave 01: Represents a vendor managing an inventory of items.
+    Attributes: inventory (list), default is an empty list.
+    Methods: add(item) adds and returns the item; 
+    remove(item) removes and returns the item or False if not found.
+    """
+
+    def __init__(self, inventory=None):
+        if not inventory:
+            self.inventory = []
+        else:
+            self.inventory = inventory
```

Comment:
You could use a ternary to write this like:
```suggestion
        self.inventory = [] if not inventory else inventory
```